haml/_header.html.haml

```diff
@@ -1,9 +1,18 @@
-- links ||= []
-- sign_in_out ||= {}
+:ruby
+  links = header.fetch("links", [])
+  sign_in_out = header.fetch("sign_in_out", {})
+  main_content_anchor_link = header.fetch("main_content_id", "#cads-main-content")
 
 %header.cads-header
   .cads-grid-container
     .cads-grid-row
+      %nav.cads-skip-links
```

Comment:
Wondering if this should be completely outside the `header` element? i.e. the first thing on the page.

Reply:
Does it make a difference? It's still the first thing to receive focus.

Reply:
Probably not, it only looked funny to see something loose within a `cads-grid-row`, or do we still want it to sit within the grid? If so then crack on!

Reply:
Yeah it needs to be in the grid 👍